tests complain about running as root if ginkgo isn't installed

I didn't have `ginkgo` installed and all I get is this misleading error:
```
# The tests need to run as root
sudo CGO_ENABLED=0 ETCD_IP=127.0.0.1 PLUGIN=calico GOPATH=/home/turk/go 
usage: sudo -h | -K | -k | -V
usage: sudo -v [-AknS] [-g group] [-h host] [-p prompt] [-u user]
usage: sudo -l [-AknS] [-g group] [-h host] [-p prompt] [-U user] [-u user] [command]
usage: sudo [-AbEHknPS] [-r role] [-t type] [-C num] [-g group] [-h host] [-p prompt] [-u user]
            [VAR=value] [-i|-s] [<command>]
usage: sudo -e [-AknS] [-r role] [-t type] [-C num] [-g group] [-h host] [-p prompt] [-u user] file
            ...
Makefile:92: recipe for target 'test' failed
make: *** [test] Error 1
```

We should at least mention in the README that ginkgo is required.

--- a/README.md
+++ b/README.md
@@ -17,11 +17,11 @@ This repository includes a top-level CNI networking plugin, as well as a CNI IPA
 To learn more about CNI, visit the [appc/cni][cni] repo.
 
 ## Building the plugins and running tests
-To build the Calico Networking Plugin for CNI locally, clone this repository and run `make`.  This will build both CNI plugin binaries and run the tests.
+To build the Calico Networking Plugin for CNI locally, clone this repository and run `make`.  This will build both CNI plugin binaries and run the tests. This requires a recent version of Docker.
 
 - To just build the binaries, with no tests, run `make binary`. This will produce `dist/calico` and `dist/calico-ipam`.
 - To only run the tests, simply run `make test`.
-
+- To run a non-containerized build (i.e. not inside a docker container) you need to have Go 1.7+ and glide installed. 
 [cni]: https://github.com/appc/cni
 
 [![Analytics](https://calico-ga-beacon.appspot.com/UA-52125893-3/calico-cni/README.md?pixel)](https://github.com/igrigorik/ga-beacon)

